app/build.gradle.kts

```diff
@@ -0,0 +1,225 @@
+import java.util.*
+
+plugins {
+    id("com.android.application")
+    id("kotlin-android")
+    id("kotlin-kapt")
+    id("kotlin-parcelize")
+    id("androidx.navigation.safeargs.kotlin")
+    id("realm-android")
+    id("io.sentry.android.gradle")
+    id("de.mannodermaus.android-junit5")
+}
+
+android {
+
+    namespace = "com.infomaniak.drive"
+
+    compileSdk = 34
+
+    ndkVersion = "28.0.13004108"
+
+    defaultConfig {
+        testInstrumentationRunnerArguments["runnerBuilder"] = "de.mannodermaus.junit5.AndroidJUnit5Builder"
+        applicationId = "com.infomaniak.drive"
+        minSdk = 24
+        targetSdk = 34
```

Comment:
compileSdk, minSdk, targetSdk & javaVersion should be declared in project build.gradle, as in SwissTransfer project.

Reply:
Did it in this PR: https://github.com/Infomaniak/android-kDrive/pull/1606